101-bakup-vm-recovery-services-vault/azuredeploy.json

```diff
@@ -0,0 +1,83 @@
+{
+
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+
+  "contentVersion": "1.0.0.0",
+
+  "parameters": {
+
+    "vaultName": {
```

Comment:
I am not sure whether it's general guidelines or not, but if we are expecting this to be already existing, should prefix with "existing", in that case variable names should be existingVaultName, existingVmResourceGroupName, existingVmName etc.